ddtrace/monkey.py

```diff
@@ -219,6 +222,8 @@ def patch_module(module, raise_errors=True):
         return False
 
 
+@deprecated(message="This module has been moved to ddtrace.internal and will no longer be part of the public API.",
```

Comment:
We probably won't move this to `ddtrace.internal`, (and it isn't a module) so maybe a message more like:

> This function will be removed

Reply:
ha literally what I had before and changed it.